exporter/opensearchexporter/index_resolver_test.go

```diff
@@ -193,13 +193,73 @@ func TestIndexResolver_ResolveTraceIndex_UnknownPlaceholder(t *testing.T) {
 
 	td := createTestTraceData("myservice")
 	ts := time.Date(2025, 6, 7, 0, 0, 0, 0, time.UTC)
-	index := resolver.ResolveTraceIndex(cfg, td, ts)
+	index := resolver.ResolveTraceIndex(cfg, td.ResourceSpans().At(0), ts)
 	expected := "otel-traces-unknown-2025.06.07"
 	if index != expected {
 		t.Errorf("expected %q, got %q", expected, index)
 	}
 }
 
+func TestIndexResolver_ResolveLogIndex_MultipleResources(t *testing.T) {
+	resolver := newIndexResolver()
+	cfg := &Config{
+		LogsIndex:           "%{service.name}-logs",
```

Comment:
What if the index name is not a resource attribute but a scope or log/span attribute? In that case, the logic would still fail and select only the last received value, right?